liquidctl/driver/corsair_hid_psu.py

```diff
@@ -67,6 +67,9 @@ class CorsairHidPsu(UsbHidDriver):
 
     # support for hwmon: corsair-psu, Linux 5.11 (5.13 recommended)
 
+    # note: quadratic equation coefficients calculated with the following script:
+    # https://github.com/liquidctl/collected-device-data/blob/master/psu-efficiency/interpolate
```

Comment:
I calculated the coefficients manually before I found this from the previous PR... oh well